modules/networking/private_dns_resolvers_forwarding_rules/module.tf

```
@@ -0,0 +1,30 @@
+resource "azurecaf_name" "pvtdnsrfr" {
+  name          = var.settings.name
+  resource_type = "azurerm_private_dns_resolver_forwarding_rule"
+  prefixes      = var.global_settings.prefixes
+  random_length = var.global_settings.random_length
+  clean_input   = true
+  passthrough   = var.global_settings.passthrough
+  use_slug      = var.global_settings.use_slug
+
+}
+
+
+resource "azurerm_private_dns_resolver_forwarding_rule" "pvt_dns_resolver_forwarding_rule" {
+  name                      = azurecaf_name.pvtdnsrfr.result
+  dns_forwarding_ruleset_id = var.dns_forwarding_ruleset_id
+  domain_name               = var.settings.domain_name
+  enabled                   = can(var.settings.enabled) ? try(var.settings.enabled, true) : true
```

Let the provider set the default value
```suggestion
  enabled                   = try(var.settings.enabled, null)
```